Description: Remove git in gemspec
 git is not used in debian build system
Last-Update: 2021-11-22

Index: ruby-ffi/ffi.gemspec
===================================================================
--- ruby-ffi.orig/ffi.gemspec
+++ ruby-ffi/ffi.gemspec
@@ -16,18 +16,8 @@ Gem::Specification.new do |s|
     s.metadata['source_code_uri'] = 'https://github.com/ffi/ffi/'
     s.metadata['mailing_list_uri'] = 'http://groups.google.com/group/ruby-ffi'
   end
-  s.files = `git ls-files -z`.split("\x0").reject do |f|
-    f =~ /^(\.|bench|gen|libtest|nbproject|spec)/
-  end
-
-  # Add libffi git files
-  lfs = `git --git-dir ext/ffi_c/libffi/.git ls-files -z`.split("\x0")
-  # Add autoconf generated files of libffi
-  lfs += %w[ compile configure config.guess config.sub install-sh ltmain.sh missing fficonfig.h.in ]
-  # Add automake generated files of libffi
-  lfs += `git --git-dir ext/ffi_c/libffi/.git ls-files -z *.am */*.am`.gsub(".am\0", ".in\0").split("\x0")
-  s.files += lfs.map do |f|
-    File.join("ext/ffi_c/libffi", f)
+  s.files = Dir.glob('**/*').reject do |f|
+    f =~ /^(\.|bench|gen|libtest|nbproject|spec|debian)/
   end
 
   s.extensions << 'ext/ffi_c/extconf.rb'
